src/main/java/com/actiontech/dble/manager/handler/DatabaseHandler.java

```diff
@@ -63,48 +66,66 @@ public static void handle(String stmt, ManagerConnection c) {
                 return;
             }
         }
-        final List<String> errMsg = new CopyOnWriteArrayList<>();
+
+        final List<String> errDataNodes = new CopyOnWriteArrayList<>();
+        final Map<String, PhysicalDBNode> allDataNodes = DbleServer.getInstance().getConfig().getDataNodes();
         final AtomicInteger numberCount = new AtomicInteger(dataNodes.size());
         for (final String dataNode : dataNodes) {
-            PhysicalDBNode dn = DbleServer.getInstance().getConfig().getDataNodes().get(dataNode);
+            PhysicalDBNode dn = allDataNodes.get(dataNode);
             final PhysicalDatasource ds = dn.getDbPool().getSource();
             final String schema = dn.getDatabase();
             OneRawSQLQueryResultHandler resultHandler = new OneRawSQLQueryResultHandler(new String[0], new SQLQueryResultListener<SQLQueryResult<Map<String, String>>>() {
                 @Override
                 public void onResult(SQLQueryResult<Map<String, String>> result) {
                     if (!result.isSuccess()) {
-                        errMsg.add(dataNode);
+                        errDataNodes.add(dataNode);
+                    } else if (isCreate) {
+                        tryResolve(ds.getHostConfig().getName(), ds.getConfig().getHostName(), dataNode, schema, ds.getConfig().getId());
```

Comment:
after creating databases success, PhysicalDBNode database exists flag should be changed with